### ISSUE TITLE
test set

### ISSUE THREAD

**Issue by yyy010119** [ShiratsuyuShigure/MLAENet-pytorch#5]
Hello, may I ask about the specific distribution of your dataset's test set, training set, and validation set?
